code/game/objects/items/implants/implantuplink.dm

```diff
@@ -28,6 +29,10 @@
 	name = "implanter (uplink)"
 	imp_type = /obj/item/implant/uplink
 
+/obj/item/implanter/uplink/Initialize(mapload, uplink_flag = UPLINK_TRAITORS)
+	imp = new imp_type(src, null, uplink_flag)
```

Comment:
Is it possible to use named arguments here?

Reply:
It does not seem to be possible.